storage/country.cpp

```diff
@@ -100,15 +122,21 @@ class DoStoreCountries
 class DoStoreFile2Info
 {
   map<string, CountryInfo> & m_file2info;
-  string m_lastFlag;
+  int64_t const m_version;
 
 public:
-  DoStoreFile2Info(map<string, CountryInfo> & file2info) : m_file2info(file2info) {}
+  DoStoreFile2Info(map<string, CountryInfo> & file2info, int64_t version)
+    : m_file2info(file2info), m_version(version) {}
 
-  void operator()(string name, string file, string const & flag, uint32_t mapSize, uint32_t, int)
+  void operator()(string name, string file, uint32_t mapSize, uint32_t, int)
   {
-    if (!flag.empty())
-      m_lastFlag = flag;
+    if (version::IsSingleMwm(m_version))
```

Comment:
Since we need to support two different courtries.txt formats. Did I answer your question?